genome.spec

```diff
@@ -107,14 +178,59 @@ typedef structure {
 } CDS;
 
 /*
-Structure for a single feature mRNA
+The mRNA is the transcribed sequence from the original feature, minus the
+introns, but including the UTRs.
 
-      flags are flag fields in GenBank format. This will be a controlled vocabulary.
-        Initially Acceptable values are pseudo, ribosomal_slippage, and trans_splicing
-        Md5 is the md5 of dna_sequence.
+Fields:
+  id: string - identifying string for the mRNA
+  location: list of (string, int, string, int) - list of locations from where
```

Comment:
This is a case where it might make sense to define a location type so the documentation can be centralized

Reply:
I agree, but I want to minimize actual type changes for this update